tools/pd-simulator/simulator/raft.go

```diff
@@ -305,3 +344,63 @@ func generateSplitKey(start, end []byte) []byte {
 	key = append(key, ('a'+'z')/2)
 	return key
 }
+
+func mustDecodeMvccKey(key []byte) []byte {
+	left, res, err := table.DecodeBytes(key)
+	if len(left) > 0 {
+		simutil.Logger.Fatalf("Decode key left some bytes: %v", key)
+	}
+	if err != nil {
+		simutil.Logger.Fatalf("Decode key %v meet error: %v", res, err)
+	}
+	return res
+}
+
+func generateMvccSplitKey(start, end []byte) []byte {
```

Comment:
emm, add a description about what is it different from generateSplitKey and how it works?

Reply:
old way not support the encode key and has a bug like `generateSplitKey("a","ab")`, will get “am”